src/Context/Node/NodeContext.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @author Sergii Bondarenko, <sb@firstvector.org>
+ */
+namespace Drupal\TqExtension\Context\Node;
+
+use Drupal\TqExtension\Context\RawTqContext;
+
+class NodeContext extends RawTqContext
+{
+    /**
+     * @When /^I (visit|view|edit) "([^"]+)" node of type "([^"]+)"$/
+     */
+    public function iVisitNodePageOfType($operation, $title, $type) {
+        if ($operation == 'visit') {
+            $operation = 'view';
+        }
+        $query = new \EntityFieldQuery();
```

Comment:
we know the title upfront when we create nodes, no need to guess it, so I still disagree. We never should rely on existing content on our tests, so we create our own unique nodes and test for their titles. Guessing brings no value when you have control over content you create and want to view.